### ISSUE TITLE
letschat taiga and sonarqube apps are missing the provider=fabric8 label

### ISSUE THREAD

**Issue by rawlingsj** [fabric8io/fabric8#4214]
If you want to do a  `oc delete all -l provider=fabric8` then you are still left with the apps above

**Comment by jimmidyson** [comment]
The provider is added by the fabric8 maven plugin when generating the spec. Are these handcrafted specs?

**Comment by rawlingsj** [comment]
Yeah they are but only for now, I'm in the process of converting them to Java.  Will leave this open to capture the converting to generated specs.
